packages/astro-embed/README.md

```diff
@@ -65,6 +65,9 @@ export default defineConfig({
 });
 ```
 
+> [!IMPORTANT]
+> The `embeds()` integration must be listed before the `mdx()` integration.
```

Comment:
Thanks @mattrossman!

One question: does this format render the same way when the README is published to npm? If not, perhaps we can just incorporate this into the sentence immediately before the code block, something like:

```md
To enable the integration, add it to the `integrations` array in your `astro.config.mjs` file before the `mdx()` integration:
```

Reply:
Good point, based on what I see for `feather-icons` it appears no.

Github:
<img width="612" alt="CleanShot 2023-10-20 at 16 17 44@2x" src="https://github.com/delucis/astro-embed/assets/22670878/8cfbaf15-0ea1-469e-af4d-1a5ce3a466f0">

NPM:
<img width="808" alt="CleanShot 2023-10-20 at 16 17 36@2x" src="https://github.com/delucis/astro-embed/assets/22670878/6df9da3b-557f-4f98-b7af-23df9a798495">

Just changed this to your suggestion.